pushy/src/main/java/com/eatthepath/pushy/apns/ApnsChannelPool.java

```diff
@@ -152,10 +152,12 @@ private void acquireWithinEventExecutor(final Promise<Channel> acquirePromise) {
                         ApnsChannelPool.this.allChannels.add(channel);
                         ApnsChannelPool.this.metricsListener.handleConnectionAdded();
 
+                        log.debug("Created channel {}", channel);
```

Comment:
Not sure how often this happens, but maybe it's infrequent enough for info?

Reply:
Thank you for the feedback. For context, please see the [logging guidelines in the README](https://github.com/jchambers/pushy#logging). I don't think this qualifies as an "important lifecycle event," and so am confident `debug` is the right level for this one.

Reply:
Sounds good.